components/wsdl-tools/wsdlvalidator/org.wso2.carbon.tools.wsdlvalidator/src/main/java/org/wso2/carbon/tools/wsdlvalidator/WsdlValidator.java

```diff
@@ -230,8 +234,15 @@ private URL loadXMLToFile(Document document) throws TransformerException, IOExce
         tempFile.deleteOnExit();
         FileWriter writer = new FileWriter(tempFile);
         StreamResult result = new StreamResult(writer);
-        TransformerFactory transformerFactory = TransformerFactory
-                .newInstance("com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl", null);
+        TransformerFactory transformerFactory;
+        try {
+            transformerFactory = TransformerFactory
+                    .newInstance("com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl", null);
+        } catch (NoSuchMethodError e) {
+            logger.log(Level.INFO, "TransformerFactory.newInstance(String, ClassLoader) method not found. " +
```

Comment:
This is using another method if a method do not exists... I do not think it is a warning or an error